.travis.yml

```diff
@@ -1,30 +1,23 @@
 sudo: false
 language: python
-python:
-    - "2.7"
-    - "3.4"
```

Comment:
This unfortunately won't work. Every entry under the 'python' key creates another matrix of tests. If I added both Python 3.5 and 2.7 here we'd run the tests twice, duplicating them. See:
https://docs.travis-ci.com/user/customizing-the-build/#Build-Matrix

(And yeah Travis makes python2.7, python3.4 available by default, as I mentioned in the commit message :-))